unime/tailwind.config.js

```diff
@@ -15,6 +15,9 @@ export default {
         'text-alt': 'rgb(var(--color-text-alt))',
         background: 'rgb(var(--color-background))',
         'background-alt': 'rgb(var(--color-background-alt))',
+        // `gray` instead of `grey` to avoid conflict with old color definition `grey`.
```

Comment:
If we intend to resolve this issue of color names, please mark it as a `TODO`.

Reply:
Obsolete. We don't need `gray` and `gray-alt`.